assets/js/comments.js

```diff
@@ -0,0 +1,174 @@
+var video_data = JSON.parse(document.getElementById('video_data').textContent);
+
+var spinnerHTML = '<h3 style="text-align:center"><div class="loading"><i class="icon ion-ios-refresh"></i></div></h3>';
+var spinnerHTMLwithHR = spinnerHTML + '<hr>';
+
+String.prototype.supplant = function (o) {
+    return this.replace(/{([^{}]*)}/g, function (a, b) {
+        var r = o[b];
+        return typeof r === 'string' || typeof r === 'number' ? r : a;
+    });
+};
+
+function toggle_comments(event) {
+    var target = event.target;
+    var body = target.parentNode.parentNode.parentNode.children[1];
+    if (body.style.display === 'none') {
+        target.textContent = '[ − ]';
+        body.style.display = '';
+    } else {
+        target.textContent = '[ + ]';
+        body.style.display = 'none';
+    }
+}
+
+function hide_youtube_replies(event) {
+    var target = event.target;
+
+    var sub_text = target.getAttribute('data-inner-text');
+    var inner_text = target.getAttribute('data-sub-text');
+
+    var body = target.parentNode.parentNode.children[1];
+    body.style.display = 'none';
+
+    target.textContent = sub_text;
+    target.onclick = show_youtube_replies;
+    target.setAttribute('data-inner-text', inner_text);
+    target.setAttribute('data-sub-text', sub_text);
+}
+
+function show_youtube_replies(event) {
+    var target = event.target;
+
+    var sub_text = target.getAttribute('data-inner-text');
+    var inner_text = target.getAttribute('data-sub-text');
+
+    var body = target.parentNode.parentNode.children[1];
+    body.style.display = '';
+
+    target.textContent = sub_text;
+    target.onclick = hide_youtube_replies;
+    target.setAttribute('data-inner-text', inner_text);
+    target.setAttribute('data-sub-text', sub_text);
+}
+
+function get_youtube_comments() {
+    var comments = document.getElementById('comments');
+
+    var fallback = comments.innerHTML;
+    comments.innerHTML = spinnerHTML;
+
+    var baseUrl = video_data.base_url || '/api/v1/comments/'+ video_data.id
+    var url = baseUrl +
+        '?format=html' +
+        '&hl=' + video_data.preferences.locale +
+        '&thin_mode=' + video_data.preferences.thin_mode;
+
```

Comment:
There are some trailing whitespaces here!

Reply:
I'm not sure what you mean?

Reply:
download https://github.com/iv-org/invidious/pull/4010.diff and check around L153 ^^

![image](https://github.com/iv-org/invidious/assets/52980486/8eb2ff80-d74a-4473-b9a4-8f21eec90581)

Reply:
Ahhh okay, I think you commented on the wrong line lol. Should be fixed now 😄